.github/workflows/build-runner-images.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
         with:
           params: |
-            AMI_ACCOUNT=/gold-image/account
-            AMI_FILTER=/gold-image/filter
+            AMI_ACCOUNT=/github-actions-runner-ami/account
+            AMI_FILTER=/github-actions-runner-ami/filter
```

Comment:
Looks good but would you mind setting these at `/github-runner/gold-image-account` and `/github-runner/gold-image-filter` to match the path I set up for other github-runner params?